app/src/main/java/org/cirdles/topsoil/app/util/StandardGitHubIssueCreator.java

```diff
@@ -0,0 +1,38 @@
+package org.cirdles.topsoil.app.util;
+
+import org.cirdles.topsoil.app.browse.WebBrowser;
+import org.cirdles.topsoil.app.metadata.ApplicationMetadata;
+
+import javax.inject.Inject;
+
+/**
+ * Created by Benjam on 11/20/2015.
+ */
+public class StandardGitHubIssueCreator extends GitHubIssueCreator {
+
+    private final ApplicationMetadata metadata;
+
+    @Inject
+    public StandardGitHubIssueCreator(
+            ApplicationMetadata metadata,
+            WebBrowser browser) {
+        super(browser);
+        this.metadata = metadata;
+
+        printHeader();
+    }
+
+    private void printHeader() {
+
+        String topsoilVersion = metadata.getVersion();
+        String javaVersion = System.getProperty("java.version");
+        String operatingSystem = System.getProperty("os.name") + " " + System.getProperty("os.version");
+
+        println("Topsoil Version: " + topsoilVersion);
+        println("Java Version: " + javaVersion);
+        println("Operating System: " + operatingSystem);
+        println("*****");
```

Comment:
Use either `***`, `___`, or `---` here, because it'll render the same in GitHub, and it saves a few characters.